HelloSoni.txt

```diff
@@ -1,4 +1,5 @@
 {
 	"name":"Soni Saw"
 	"age":32
+	"address":"B1-424"
```

Comment:
why this change?

Reply:
Hey this is adding as part of USxxx where UI has been enhanced to have address of user.